zhaquirks/tuya/ts0601_motion.py

```diff
@@ -0,0 +1,77 @@
+"""Quirk for the Mercator Ikuü TS0601 motion sensor (_TZE200_agumlajc)."""
+
+from zigpy.quirks.v2 import EntityType
+from zigpy.quirks.v2.homeassistant import UnitOfTime
+from zigpy.quirks.v2.homeassistant.binary_sensor import BinarySensorDeviceClass
+from zigpy.quirks.v2.homeassistant.sensor import SensorDeviceClass
+import zigpy.types as t
+
+from zhaquirks.tuya.builder import TuyaQuirkBuilder
+
+
+class LightControlMode(t.enum8):
+    """Enum for the backlight mode setting (dp_id=105)."""
+
+    On = 0
+    Off = 1
+    Auto = 2
+
+
+# Quirk definition for the Mercator Ikuu Combination Sensor (_TZE200_agumlajc)
+(
+    TuyaQuirkBuilder("_TZE200_agumlajc", "TS0601")
+    .applies_to("_TZE204_agumlajc", "TS0601")
+    .tuya_temperature(dp_id=1, scale=10)
+    .tuya_humidity(dp_id=2)
+    .tuya_illuminance(dp_id=101)
+    .tuya_number(
+        dp_id=102,
+        attribute_name="brightness_threshold",
+        type=t.uint16_t,
+        device_class=SensorDeviceClass.ILLUMINANCE,
+        min_value=0,
+        max_value=1000,
+        step=1,
+        translation_key="brightness_threshold",
+        fallback_name="Brightness Threshold",
```

Comment:
Due to HA naming conventions, please only capitalize the first word, i.e. this should be "Brightness threshold" for the `fallback_name` (which will be the basis for US English translations when it's merged into HA).
Please also change the other names.